### ISSUE TITLE
Base – Assembly_E4

### ISSUE THREAD

**Issue by PabloAbara** [IIC2343-2016-2/syllabus#53]
Hola, 
Aún no podemos generar la ROM  del ejemplo en la entrega pero; 
1) Entiendo que lo que debería hacer es mostrar "flucchini@uc.cl" en la pantalla LCD. Es correcto? 

2) Con respecto a la librería, se debe usar algo de 'dentro' de la librería? O solo de CALL lcd_char / CALL lcd_string para mostrar?

Gracias!

**Comment by FranLucchini** [comment]
1. Exactamente, debe mostrar eso.
2. Bueno, en lo que más se deben enfocar es usar el método `std_io_btn_wait` (espera que se presione y se de de presionar el botón central) y todo lo que está en el bloque Libreria lcd, que incluye las que mencionaste. Para poder entender qué hacen los métodos lcd, deben leer el manual de la placa. En él aparecen los comandos que se le pueden enviar a la lcd.

**Comment by PabloAbara** [comment]
Dale Gracias!! 

¿ El método `std_io_btn_wait` sólo aplica al botón central ?

**Comment by FranLucchini** [comment]
No, de hecho si lo revisan guarda en el regA el "estado" de los botones y luego lo ingresa ese estado en regB para comparar y ver si hay un cambio.

**Comment by fagomez2** [comment]
Hola! ¿Cómo guarda ese estado de los botones? ¿Acaso se utilizan 1's o 0's para cada botón en un orden en particular? Gracias.

**Comment by FranLucchini** [comment]
Exactamente, la entrada de los botones es un arreglo de bits y cada elemento del arreglo corresponde a un botón.
